notes/README.md

```diff
@@ -0,0 +1,592 @@
+| sendAmount | Amount | The amount that will be debited from the account. (Optional)
+| fundsAreAvailable | Boolean | Indicates that there are funds available in the underlying account to complete this payment. This is not a guarantee they will still be available when the payment is attempted but it is a good indicator to a client that it should attempt to authorize the payment. (Protected)
+| expiresAt | DateTime | The date and time when the calculated send amount is no longer valid. (Read Only)
+| description | string | Human readable description of the outgoing payment. (Optional)
+| externalRef | string | A reference that can be used by external systems to reconcile this payment with their systems. E.g. An invoice number or purchase order number. (Optional)
+| receiptNonce | uint128 | A random nonce to identify the underlying ILP STREAM connection used to send this payment. (Optional)
+| receiptSecret | uint256 | A secret used to sign receipts returned on the ILP STREAM connection used to send this payment. (Optional)
+
+#### 3.2 Actions
+
+Actions are executed against the base URL of the Outgoing Payment resource.
+
+#### 3.2.1 Request Interledger Payment Receipts
+
+**Method**: GET
+**Accept**: application/ilp-stream-receipt-v1+json
+
+Returns the latest STREAM Receipt received while executing this Outgoing Payment. 
+
+#### 3.2.2 Create Outgoing Payment
+
+**Method**: POST
+**Content-Type**: application/op-outgoing-payment-v1+json
+**Accept**: application/op-outgoing-payment-v1+json
+
+Create a new Outgoing Payment sub-resource.
+
+The request body contains an Outgoing Payment resource encoded as JSON with only the following properties:
+
+| Property | Required | Notes
+| -- | -- | --
+| state | No | Only a value of authorized is permitted and only if the client has permission to authorize outgoing payments.
+| receivingAccount | Maybe | Either a receivingAccount or a receivingPayment MUSt be provided.
+| receivingPayment | Maybe | Either a receivingAccount or a receivingPayment MUSt be provided.
+| receiveAmount | Maybe | Required if no sendAmount and no receivingPayment is provided. assetCode and assetScale MUST match those of the receivingAccount.
+| sendAmount | Maybe | Required if no receiveAmount and no receivingPayment is provided.  assetCode and assetScale MUST match those of the underlying Account.
+| description | No | A description or message that will be available to the sender and MAY be used by the provider if creating an incoming payment at the receivingAccount.
+| externalRef | No | An external reference number or code that can be used to reconcile the payment with external systems such as an invoice number.
+| receiptNonce | No | A random nonce to identify the underlying ILP STREAM connection used to send this payment. (Optional)
+| receiptSecret | No | A secret used to sign receipts returned on the ILP STREAM connection used to send this payment. (Optional)
```

Comment:
```suggestion
```
Generally the sender shouldn't have access to the receipt secret, otherwise they could produce bogus receipts.
Is there a use case for this being specified in the outgoing payment?